coreneuron/nrniv/output_spikes.cpp

```diff
@@ -95,8 +95,12 @@ void local_spikevec_sort(std::vector<double>& isvect,
 #if NRNMPI
 
 void sort_spikes(std::vector<double>& spikevec_time, std::vector<int>& spikevec_gid) {
-    double lmin_time = *(std::min_element(spikevec_time.begin(), spikevec_time.end()));
-    double lmax_time = *(std::max_element(spikevec_time.begin(), spikevec_time.end()));
+    double lmin_time = std::numeric_limits<int>::max();
```

Comment:
ok well, I do have one more question. Why do you use `std::numeric_limits<int>` and not `std::numeric_limits<double>` ?

Reply:
You are right, I will change this.